src/Core/Util/String/StringModifierInterface.php

```diff
@@ -47,4 +47,11 @@ public function splitByCamelCase($string);
      * @return string
      */
     public function cutEnd(string $string, int $expectedLength): string;
+
+    /**
+     * @param string $string
+     *
+     * @return string
+     */
+    public function convertCamelCaseToKebabCase(string $string): string;
```

Comment:
What is the reason to prefer kebab case and not use https://github.com/symfony/serializer/blob/3.4/NameConverter/CamelCaseToSnakeCaseNameConverter.php

Reply:
Reason is that md files in prestashop docs follows kebab case convention

Reply:
It follows kebab case because of Hugo, here we can use whatever we want :)

Reply:
These files will be outputed straight into docs project (that was the idea) so why shouldn't we support hugo?